backtrace-library/src/main/cpp/backends/crashpad-backend.cpp

```diff
@@ -256,7 +353,8 @@ void ReEnableCrashpad() {
     // Re-enable uploads if disabled
     if (disabled) {
         if (database == nullptr) {
-            __android_log_print(ANDROID_LOG_ERROR, "Backtrace-Android", "Crashpad database is null, this should not happen");
```

Comment:
Seems like these hunks should be moved back to the NFC formatting commit. It would make it easier to determine what actually needs to be reviewed.